src/spi.rs

```diff
@@ -240,12 +254,12 @@ impl<'d> SpiBusMasterDriver<'d> {
         }
 
         match read.len().cmp(&write.len()) {
-            Ordering::Equal => { /* Nothing left to do */ }
-            Ordering::Greater => {
+            cmp::Ordering::Equal => { /* Nothing left to do */ }
```

Comment:
Why `cmp::`?

Reply:
in my first attempt using the critical section i used AtomicBool -> also needed Ordering for that. Not anymore can revert here